battleship-server/src/main/java/com.epam.solid.nie/server/ServerThread.java

```diff
@@ -8,9 +8,11 @@
 
 class ServerThread extends Thread{
     private final Socket clientSocket;
+    private final int id;
```

Comment:
the same thing like in Protocol interface. Why we need this id?

Reply:
removed in last commit